Tutorials/Animations_with_xmovie.ipynb

```diff
@@ -0,0 +1,1515 @@
+{
```

Comment:
<p>What about adding <code>overwrite_existing=True</code> ? Otherwise this will throw an error if the file already exists and the testing might have trouble?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/452/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
I was thinking about it, but also I thought it's better that the default does not have the `overwrite_existing=True` so that people don't accidentally delete something they didn't want to. We can add it later if this creates issue for tests?